src/librustc_error_codes/error_codes/E0759.md

```diff
@@ -5,14 +5,11 @@ Erroneous code examples:
 ```compile_fail,E0759
 use std::fmt::Debug;
```

Comment:
Does this still needs `Debug`? I saw that the rest have the `use` removed.

Reply:
;)